src/features/Issues/Issues.js

```diff
@@ -3,7 +3,9 @@ import "./Issues.scss";
 import { Link } from "react-router-dom";
 import { connect } from "react-redux";
 import { getIssues, filterIssues } from "./actions";
-import Icon from "./components/Icon/Icon";
+import Icon from "../Icon/Icon";
```

Comment:
Woops, should be the other import instead

Reply:
![image](https://user-images.githubusercontent.com/16860528/46896759-ee8f3880-ce4c-11e8-8eaa-19421896c800.png)